### DISCUSSION TITLE
can shadodsocks bypass censorship?

### DISCUSSION THREAD

**Discussion by aria1346** [shadowsocks/shadowsocks-org#209] [Q&A]
Hello community,

I want to know if shadowsocks is more sophisticated than normal vpn available on global market? I am living in IRAN and Government censors media and social websites and apps like facebook, instagram, telegram, whatsapp, skype etc, even we use all available free vpns on global market none of them work. is shadowsocks much more sophisticated than VPN protocols that can help us bypass censorship and enables us have access to free internet?

Thanks

**Comment by Na2195** [comment]
سلام 
اصلا دشوار نیست

**Reply by aria1346** [reply]
سلام
ممنون از جوابتون . شما فارسی نوشتید من هم فارسی می نویسم.  منظورتون اینکه دشوار نیست یعنی راه اندازی سرور shadowsocks  راحته؟ضمنا چند روز پیش در کانال های اطلاع رسانی در مورد اوتلاین outline خوندم هرچی کدهای رایگان در سایت اوتلاین و کانال های تلگرامی دیدم امتحان کردم اصلا وصل نشدم و دارم به اوتلاین هم فکر می کنم که سرور اوتلاین داشته باشم. اما آیا اوتلاین از لحاظ کارایی مثل shadowsocks است یا با هم تفاوت دارن؟

ممنون

**Comment by fisherwei** [comment]
hi 伊朗老铁,

shadowsocks was designed to specifically fuck China Great Firewall, maybe you know, your country gets a lot of help from China to block Internet. So, your question is YES.

shadowsocks is very lightweight than VPN. because it is essentially a SOCKS proxy server, unlike a VPN which is a tunnel. You just need to start one process on the server and another process on your PC/Phone.

Good luck.
